src/styles/_vars.scss

```diff
@@ -9,3 +9,6 @@ $kongponentsKongIconSelector: ".kui-icon";
 
 // transition duration and transition timing function
 $kongponentsTransitionDurTimingFunc: var(--kui-animation-duration-20, $kui-animation-duration-20) ease-in-out;
+
+// KCopy tooltip selector
```

Comment:
> I'm wondering if this should be a JavaScript constant instead of a SCSS variable

I think I can't use JS constant as a selector in style block, can I?